src/mon_apl.js

```diff
@@ -673,41 +673,8 @@
         if (t === 'for' || t === 'while' || t === 'repeat') {
           suggestions.push(...['Continue', 'Leave'].map(textItem));
         }
-        if (t === 'class' || t === 'namesapce') {
```

Comment:
I'm pretty sure `namesapce` must have been a typo. However, I'm not well-versed enough in APL to know where properties belong. In my mind, we should remove the namespace for this `if` and only show the `:Property` autocomplete within a class

Reply:
Great, thanks for the feedback @e9gille. I've done 3 things now:

1. Moved `:Class`, `:Namespace` and `:Interface` back within the original `if` block, now with the fixed typo.

2. Moved `:Property` within a new `if` block that only checks if we're within a `:Class` definition.

3. Moved `:Section` to the default autocomplete. If I read the docs right [here](https://docs.dyalog.com/latest/Dyalog%20Programming%20Reference%20Guide.pdf) (p.104), `:Section` are purely for readability and are allowed within both function and class definitions. 

Please forgive me, if my accommodations of your feedback are incorrect.